apps/files/src/default.js

```diff
@@ -20,8 +20,8 @@ const appInfo = {
     {
       app: 'files-version',
       component: FileInfoVersions,
-      enabled (capabilities) {
-        return !!capabilities.core
+      enabled (capabilities, highlightedFile) {
```

Comment:
for the future - we might want to do this with an object {capabilties, highlightedFile} - I think this will be extended in the future ...